localstack/services/apigateway/context.py

```diff
@@ -132,6 +132,13 @@ def auth_identity(self) -> Optional[Dict]:
                 self.auth_info["identity"] = {}
             return self.auth_info["identity"]
 
+    @property
+    def authorizer_type(self) -> str:
+        if isinstance(self.auth_info, dict):
+            if self.auth_info.get("authorizer_type") is None:
+                self.auth_info["authorizer_type"] = {}
```

Comment:
We're setting the `authorizer_type` to `{}` here, although the type hint indicates that a string gets returned. Should we simply return `self.auth_info.get("authorizer_type")` from the method here?

Reply:
Yep, should be string not `dict`